README.md

```diff
@@ -41,28 +49,31 @@ module "event_notification" {
 ```
 
 ### Required IAM access policies
+{: #EN-access}
 
-You need the following permissions to run this module.
```

Comment:
You might need to ask the devs but I think the majority of this file is generated so editing these strings will likely get overwritten next time it's generated

Reply:
The IAM access is not generated, so this is ok.